spec/requests/task_search_by_combination_spec.rb

```diff
@@ -67,8 +67,13 @@
         deadline: Time.zone.parse('2016-01-01'),
       ),
       create_task(
+        title: '未完了,期限=2016-01-02,タグ=なし,サインアップ=user_a',
+        author_id: user_a.id, assignees: [user_a],
+        deadline: Time.zone.parse('2016-01-02'),
```

Comment:
__[rubocop]__
* Style/TrailingComma: Avoid comma after the last parameter of a method call, unless each item is on its own line. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)